doc/level1.rst

```diff
@@ -726,19 +728,34 @@ keys instead.  The MUA MUST be capable of assembling these keys into
 an OpenPGP certificate (:rfc:`RFC 4880 "Transferable Public
 Key"<4880#section-11.1>`) that indicates these capabilities.
 
+Secret key protection at rest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 The secret key material should be protected from access by other
 applications or co-tenants of the device at least as well as the
 passwords the MUA retains for the user's IMAP or SMTP accounts.
 
-Secret key protection at rest
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The MUA SHOULD NOT protect the secret key with a password. All
-encrypted outgoing messages MUST be signed, which would require the
-user to enter the password for both reading and sending mail. This
-introduces too much friction to become part of a routine daily workflow.
-Protection of the user's keys at rest and other files is achieved more
-easily and securely with full-disk encryption.
+The MUA MAY protect the secret key (and other sensitive data it has
+access to) with a password, but it SHOULD NOT require the user to
+enter the password each time they send or receive a mail. Since
+Autocrypt-enabled MUAs :ref:`sign all encrypted outgoing
+messages<mail-encryption>`, it could happen that the user has to enter
+the password for both reading and sending mail. This introduces too
```

Comment:
"has to enter the password very often, both for reading and sending mail"?

Reply:
this kind of change is unobjectionable -- please add it as a patch on top of the existing series.

Reply:
I've pushed it myself now.